sdk/python/feast/type_map.py

```diff
@@ -438,8 +439,14 @@ def pa_to_value_type(pa_type: object):
 
 
 def pa_to_feast_value_type(value: Union[pa.lib.ChunkedArray, str]) -> ValueType:
+    value_type = (
+        value.type.__str__() if isinstance(value, pa.lib.ChunkedArray) else value
```

Comment:
Would you mind explaining this line?

Reply:
This is moved from line 459 - basically this converts the type of a pandas column into a string representation, if the stringified version has not already been passed in.